rpm_spec/manageiq.spec.in

```diff
@@ -69,11 +69,9 @@ cd %{_builddir}
 %{__mkdir} -p %{buildroot}/etc/httpd/conf.d
 %{__mkdir} -p %{buildroot}%{app_root}/log/apache
 %{__mkdir} -p %{buildroot}%{app_root}/tmp/{,sockets,pids}
-%{__mkdir} -p %{buildroot}%{app_root}/{certs,config}
-%{__mkdir} -p %{buildroot}%{app_root}/public/pictures
-%{__chmod} 4750 %{buildroot}%{app_root}/{log,config,certs}
+%{__chmod} 6750 %{buildroot}%{app_root}/{log,config,certs}
+%{__chmod} 6750 %{buildroot}%{app_root}/data/git_repos
```

Comment:
Should these be listed under core? (They're used in both podified and appliances)

Reply:
this directory is added to the system in `manageiq-core` rpm.

Up until now, `manageiq.spec.in` has been responsible for setting up the build environment.

Reply:
ugh. english is hard.

I'll move those lines